suffuse/node_fuse.go

```diff
@@ -25,24 +24,16 @@ func (x *Elem) Access(ctx context.Context, req *f.AccessRequest) error {
 func (x *Elem) Attr(ctx context.Context, attr *f.Attr) error {
   logD("Attr", "path", x.Path)
 
-     path := x.Path
-  fi, err := path.OsLstat()
-  var a f.Attr
-
-  if err != nil {
-    subpath, cmd := splitSedSuffix(path)
-    if cmd == "" {
-      return err
+  for _, rule := range Rules {
```

Comment:
The _ isn't the first rule, it's go's ridiculous iterator index which you can't avoid. It takes the values 0, 1, 2, ... while rule goes through the Rule values.